classAddress/classAddress.py

```diff
@@ -0,0 +1,118 @@
+import validation as valid
+
+class Address:
+    __postal_code_lenght = 5
+
+    def __init__(self, address_line = 'address_line', postal_code = 10000, country = 'country', city = 'city', fax_number = 100000, phone_number = 100000):
+        self.address_line = address_line
+        self.postal_code = postal_code
+        self.country = country
+        self.city = city
+        self.fax_number = fax_number
+        self.phone_number = phone_number
+
+
+    def __str__(self):
+        return f'\n!-----------------------!\nID- {self.ID}, \naddress line - {self.address_line} street, \npostal code - {self.postal_code}, \ncountry - {self.country}, \ncity - {self.city}, \nfax number - {self.fax_number}, \nphone number - {self.phone_number} \n!-----------------------!\n'
```

Comment:
use cycle here, method for getting property by name and dictionary (model of address)